content-test/components/NewTabPage.story.js

```diff
@@ -34,8 +34,19 @@ const Container = props => (
 // itself.
 
 storiesOf("NewTabPage", module)
-  .add("Be on the lookout popup", () =>
-    <Container>
-      <NewTabPage {...mockData} page="SPOTLIGHT_STORYBOOK" dispatch={dispatch} />,
-    </Container>
+  .add("data available (this.props.isReady==true)", () => {
```

Comment:
Fixed the casing on the new stories.  The code snippet I expect to occasionally be useful to folks using storybook for development (like I do).